docs/guide/explanation/the-deployment-process.mdx

```diff
@@ -0,0 +1,53 @@
+# The deployment process
+Deployment begins via one of Zephyr Cloud's bundler plugins.
+When added to your bundler's plugin list, the plugin deploys your bundled applicatioon to a remote Zephyr Cloud instance.
```

Comment:
Application is spelled wrong

Reply:
Resolved in [7850dd0](https://github.com/ZephyrCloudIO/zephyr-documentation/pull/5/commits/7850dd018802d2a1accd9f3fa9ff9b21e2a6fd6d)